tests/acceptance/test_nominal.py

```diff
@@ -62,29 +62,29 @@ def func(*queues, name="worker"):
 def test_nominal(defer, worker):
     from .param import Param
 
-    defer("sum_task", a=5, b=7)
-    defer("sum_task_param", p1=Param(3), p2=Param(4))
-    defer("increment_task", a=3)
+    defer("sum_task", [], a=5, b=7)
```

Comment:
Ah damn. Maybe this should be the default ?

Reply:
Sorry about that :)

Reply:
I did not want to change your work :-) But yes, I can change this.